### PR TITLE
Add move-desktop-screenshots-to-trash extension

### PR DESCRIPTION
**Pull request by robotdestroy**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/move-desktop-screenshots-to-trash/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const { exec } = require('child_process');
```
Comment (joshmgross) [inline extensions/move-desktop-screenshots-to-trash/src/index.ts:4]:
```suggestion
import fs from "fs";
import os from "os";
import path from "path";
import { exec } from "child_process";
```

Comment (robotdestroy) [reply extensions/move-desktop-screenshots-to-trash/src/index.ts:4]: Tested locally and this works, thank you!

**Comment by robotdestroy** [comment]
Should be good now!

**Comment by robotdestroy** [comment]
Added preference for custom prefixes as well as a custom directory.

**Comment by pernielsentikaer** [comment]
Hi 👋

Thanks for your contribution 🔥 

It feels pretty similar to [this extension](https://www.raycast.com/chanceaclark/empty-screenshots) by @chanceaclark  which only missing one preference to do the same, right?

Could it be an idea to enhance that one instead

**Comment by robotdestroy** [comment]
Oh yeah, I didn't notice that one! Yes, I can add the prefix preference to that extension instead! Thanks.

**Comment by robotdestroy** [comment]
Moved this work here: https://github.com/raycast/extensions/pull/12358

Closing this request.

**Pull request closed without merging**